webserver/pytreedb_server/static/dl_script.py

```diff
@@ -0,0 +1,34 @@
+import os
+import urllib.request
+import shutil
+
+source_file = None  # location of the .txt file containing point cloud urls
+output_folder = None  # local folder to save the files under
+
+def download(urls: list[str], dest_folder: str):
+    """
+    Downloads point clouds (.laz) data from a list of source URLs
+
+    :param list[str] urls: list of URLs
+    :param str dest_folder: destination folder to save data
+    """
+    # create folder if it doesn't exist yet
+    if not os.path.exists(dest_folder):
+        os.makedirs(dest_folder)
+    for url in urls:
+        filename = url.split('/')[-1]
+        file_path = os.path.join(dest_folder, filename)
+        # download the file from url and save locally
+        with urllib.request.urlopen(url) as response, open(file_path, 'wb') as out_file:
+            shutil.copyfileobj(response, out_file)
+            print("saving to", os.path.abspath(file_path))
+
+# Prompt user to enter paths if not given
```

Comment:
Seems a bit unnecessary, when there is no other options to provide paths..
Either allow providing the paths as command line arguments when calling the script or remove the if-clauses (and L 5 and 6)

Reply:
Users can enter fixed paths in line5 and line6 if they wish to, without entering the paths each time they run the script. I think it does make sense